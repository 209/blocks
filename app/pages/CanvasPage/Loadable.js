/**
 * Asynchronously loads the component for CanvasPage
 */

import loadable from 'utils/loadable';

export default loadable(() => import('pages/CanvasPage/index'));
