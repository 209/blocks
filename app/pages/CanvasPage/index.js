/*
 * CanvasPage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 */

import React from 'react';
// import { FormattedMessage } from 'react-intl';
// import messages from 'pages/CanvasPage/messages';
import CanvasBoard from 'containers/CanvasBoard';

export default function CanvasPage() {
  return (
    <>
      <h2>Canvas</h2>
      <CanvasBoard />
    </>
  );
}
