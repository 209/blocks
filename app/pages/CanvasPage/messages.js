/*
 * CanvasPage Messages
 *
 * This contains all the text for the CanvasPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.CanvasPage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the CanvasPage container!',
  },
});
