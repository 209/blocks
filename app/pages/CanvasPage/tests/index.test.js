import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import CanvasPage from 'pages/CanvasPage/index';

describe('<CanvasPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <CanvasPage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
