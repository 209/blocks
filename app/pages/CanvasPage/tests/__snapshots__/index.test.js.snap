// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CanvasPage /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the CanvasPage container!
  </span>
</h1>
`;
